code/game/objects/items/weapons/cards_ids.dm

```diff
@@ -157,6 +160,14 @@ update_label("John Doe", "Clowny")
 			src.registered_name = ""
 			return
 		src.assignment = u
+		var/list/choosable_races = list()
+		for(var/speciestype in typesof(/datum/species) - /datum/species)
+			var/datum/species/S = new speciestype()

```

Comment:
don't New these.

1. Isn't there a global list of species (think the init_subtypes proc) to grab from
2. your're only using the id, which can be obtained from the path with ```choosable_races += initial(speciestype.id)``` (this does mean extending the path of speciestype to be absolute though, /datum/species/speciestype)

There's no point or need of newing these, it's a waste.